### ISSUE TITLE
xrightleftharpoons not rendered in VSCode

### ISSUE THREAD

**Issue by mastrof** [SciML/Catalyst.jl#637]
I'm on Catalyst 13.2.0, and reactions are not correctly rendered in VSCode. MWE
```
using Catalyst, Latexify
rn = @reaction_network begin
    (a,b), 0 <--> X
end
render(latexify(rn))
```
The `latexify` output renders correctly here

$$
\begin{align*}
\varnothing &\xrightleftharpoons[b]{a} \mathrm{X}  
 \end{align*}
$$

but in VSCode I cannot render `\xrightleftharpoons`. Is it an issue on my side, or something else? I have been using Catalyst in the past and I don't remember having this problem. Is there a quick fix to this?

**Comment by isaacsas** [comment]
Actually, it looks like just adding a `\require{mhchem}` into the generated LaTeXString fixes things. I could add an option to force that when calling `latexify(rn)` if you want? I don't see how to make it automatic unfortunately.

**Comment by isaacsas** [comment]
Fixed on 13.2.1.
